share/completions/pnpm.fish

```diff
@@ -0,0 +1,20 @@
+function _pnpm_completion
+  set cmd (commandline -o)
+  set cursor (commandline -C)
+  set words (count $cmd)
+
+  set completions (eval env DEBUG=\"" \"" COMP_CWORD=\""$words\"" COMP_LINE=\""$cmd \"" COMP_POINT=\""$cursor\"" pnpm completion -- $cmd)
```

Comment:
This `eval` gives me the heeby-jeebies, and this `env` is unnecessary.

I also don't like that this sets a variable called "DEBUG", and I'm not sure you *need* COMP_CWORD - our bash-compatible completion thing is [__fish_argcomplete_complete](https://github.com/fish-shell/fish-shell/blob/master/share/functions/__fish_argcomplete_complete.fish), which doesn't set it (but could!).